getnyumba/web-api
# #166730609 Setup Django project

#### Description
- Setup empty django project
#### Type of change

Please select the relevant option


#### How Has This Been Tested?


#### How to Test

Give instructions on how reviewers can verify your work
- fetch branch using `git fetch origin  chore-setup-django-166730609  and git chekout  chore-setup-django-166730609 `
- run export SECRET_KEY="anyvalue" to set a secret key
- start project using `python manage.py runserver`
- visit localhost:8080 to view application

#### Any background context you want to add

#### PIVOIT TRACKER

[Finishes #166730609]
